README.md

```diff
@@ -191,12 +191,18 @@ OPTIONS
 
   -t, --templates=templates                [default: templates] Path to the templates folder
 
+  --devfile-registry=devfile-registry      [default: https://che-devfile-registry.openshift.io/] The URL of the Devfile
```

Comment:
We're going to implement an ability to deploy Devfile/Plugin registry along with Che with an argument, like `--deploy-devfile-registry`. it would set needed URL automatically
Also, we're going to implement auto propagation of env vars that starts with `CHE_` to Che Server, it would allow to set any value to plugin/devfile registry env var.
^ It's what deploy_che.sh script does


@metlos Do you think these arguments can be still useful? Won't it be confusing to have two `--devfile-registry` and `--deploy-devfile-registry` (this can be named differently parameters?

Reply:
Well, given the fact that people are already playing with custom registries (e.g. https://github.com/eclipse/che/issues/13634), I think it is going to be useful to have the ability to preconfigure the server with a custom registry url.

Reply:
I think it makes sense to have both parameters `--devfile-registry-url` and `--deploy-devfile-registry`.